generated/nirfmxlte/nirfmxlte.proto

```diff
@@ -13,43 +13,5072 @@ option csharp_namespace = "NationalInstruments.Grpc.NiRFmxLTE";
+  AUTO_DMRS_DETECTION_ENABLED_TRUE = 1;
+}
+
+enum AutoNPuschChannelDetectionEnabled {
+  AUTO_NPUSCH_CHANNEL_DETECTION_ENABLED_FALSE = 0;
+  AUTO_NPUSCH_CHANNEL_DETECTION_ENABLED_TRUE = 1;
+}
+
+enum AutoPcfichCfiDetectionEnabled {
+  AUTO_PCFICH_CFI_DETECTION_ENABLED_FALSE = 0;
+  AUTO_PCFICH_CFI_DETECTION_ENABLED_TRUE = 1;
+}
+
+enum AutoPdschChannelDetectionEnabled {
+  AUTO_PDSCH_CHANNEL_DETECTION_ENABLED_FALSE = 0;
+  AUTO_PDSCH_CHANNEL_DETECTION_ENABLED_TRUE = 1;
+}
+
+enum AutoResourceBlockDetectionEnabled {
+  AUTO_RESOURCE_BLOCK_DETECTION_ENABLED_FALSE = 0;
+  AUTO_RESOURCE_BLOCK_DETECTION_ENABLED_TRUE = 1;
+}
+
+enum ChpAveragingEnabled {
+  CHP_AVERAGING_ENABLED_FALSE = 0;
+  CHP_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum ChpAveragingType {
+  CHP_AVERAGING_TYPE_RMS = 0;
+  CHP_AVERAGING_TYPE_LOG = 1;
+  CHP_AVERAGING_TYPE_SCALAR = 2;
+  CHP_AVERAGING_TYPE_MAXIMUM = 3;
+  CHP_AVERAGING_TYPE_MINIMUM = 4;
+}
+
+enum ChpIntegrationBandwidthType {
+  CHP_INTEGRATION_BANDWIDTH_TYPE_SIGNAL_BANDWIDTH = 0;
+  CHP_INTEGRATION_BANDWIDTH_TYPE_CHANNEL_BANDWIDTH = 1;
+}
+
+enum ChpNoiseCalibrationDataValid {
+  CHP_NOISE_CALIBRATION_DATA_VALID_FALSE = 0;
+  CHP_NOISE_CALIBRATION_DATA_VALID_TRUE = 1;
+}
+
+enum ChpRbwAutoBandwidth {
+  CHP_RBW_AUTO_BANDWIDTH_FALSE = 0;
+  CHP_RBW_AUTO_BANDWIDTH_TRUE = 1;
+}
+
+enum ChpRbwFilterType {
+  CHP_RBW_FILTER_TYPE_FFT_BASED = 0;
+  CHP_RBW_FILTER_TYPE_GAUSSIAN = 1;
+  CHP_RBW_FILTER_TYPE_FLAT = 2;
+}
+
+enum ChpSweepTimeAuto {
+  CHP_SWEEP_TIME_AUTO_FALSE = 0;
+  CHP_SWEEP_TIME_AUTO_TRUE = 1;
+}
+
+enum ComponentCarrierSpacingType {
+  COMPONENT_CARRIER_SPACING_TYPE_NOMINAL = 0;
+  COMPONENT_CARRIER_SPACING_TYPE_MINIMUM = 1;
+  COMPONENT_CARRIER_SPACING_TYPE_USER = 2;
+}
+
+enum DigitalEdgeTriggerEdge {
+  DIGITAL_EDGE_TRIGGER_EDGE_RISING = 0;
+  DIGITAL_EDGE_TRIGGER_EDGE_FALLING = 1;
+}
+
+enum DigitalEdgeTriggerSource {
+  DIGITAL_EDGE_TRIGGER_SOURCE_UNSPECIFIED = 0;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PFI0 = 1;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PFI1 = 2;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_TRIG0 = 3;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_TRIG1 = 4;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_TRIG2 = 5;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_TRIG3 = 6;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_TRIG4 = 7;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_TRIG5 = 8;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_TRIG6 = 9;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_TRIG7 = 10;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXI_STAR = 11;
+  DIGITAL_EDGE_TRIGGER_SOURCE_PXIE_DSTARB = 12;
+  DIGITAL_EDGE_TRIGGER_SOURCE_TIMER_EVENT = 13;
+}
+
+enum DownlinkAutoCellIDDetectionEnabled {
+  DOWNLINK_AUTO_CELL_ID_DETECTION_ENABLED_FALSE = 0;
+  DOWNLINK_AUTO_CELL_ID_DETECTION_ENABLED_TRUE = 1;
+}
+
+enum DownlinkChannelConfigurationMode {
+  DOWNLINK_CHANNEL_CONFIGURATION_MODE_UNSPECIFIED = 0;
+  DOWNLINK_CHANNEL_CONFIGURATION_MODE_USER_DEFINED = 1;
+  DOWNLINK_CHANNEL_CONFIGURATION_MODE_TEST_MODEL = 2;
+}
+
+enum DownlinkTestModel {
+  DOWNLINK_TEST_MODEL_TM1_1 = 0;
```

Comment:
This is a crazy tokenization scheme, but I guess this is correct?

Reply:
Yeah... this matches the corresponding enum value names in the .NET API. Looks odd but it's consistent. FWIW I think the underscores correspond to "."s in the E_UTRA test model name they're referring to. "Test model 3.1a" => `TEST_MODEL_3_1A`

Reply:
Looks to be correct, yeah. In the LV API, those underscores amidst the numbers are periods, e.g. "Test Model 1.1", "Test Model 3.2b", etc.